waspc/examples/todoApp/main.wasp

```diff
@@ -239,6 +239,14 @@ query getDate {
   fn: import { getDate } from "@src/testTypes/operations/definitions"
 }
 
+query getSerializedObjects {
+  fn: import { getSerializedObjects } from "@src/testTypes/operations/definitions"
+}
+
+route SerializationRoute { path: "/serialization", to: SerializationPage }
+page SerializationPage {
+  component: import { SerializationPage } from "@src/pages/Serialization"
+}
```

Comment:
Can we add a link to this from the user's homepage, for easier manual troubleshooting (if we keep it, more on this later)?